fail2ban/protocol.py

```diff
@@ -55,6 +55,7 @@
 ["start <JAIL>", "starts the jail <JAIL>"], 
 ["stop <JAIL>", "stops the jail <JAIL>. The jail is removed"], 
 ["status <JAIL>", "gets the current status of <JAIL>"],
+["status <JAIL> extended", "gets the current status of <JAIL> with extended info"],
```

Comment:
I am still toying with this idea... so theoretically original status is just a 'basic' status and 'extended' one should output 'basic' + 'everything else we can tell' (which now would be cymru).  With current implementation it would be only cymru.  What if instead of adding a new public method per each such additional status info it would be just a kwarg "flavor" which could be "basic" (default), "extended", "cymru" and then implemented within ```status```?

btw -- could you provide an example of current output?